scripts/tasks/4_ICA.ipynb

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
<p>That could probably be converted to a pandas dataframe and saved with pandas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
sure, though less convenient for the user? (for sure cleaner if we need to return there)
The plan was so that the user could open/view the dataframe in an easy way, then compare it to the topomap view of the ICA components.

Right now this would mean saving an excel or csv file then opening it in excel, vs opening a notepad/text editor?